src/app/tabs/run-tab.js

```diff
@@ -421,24 +422,42 @@ function contractDropdown (events, self) {
       return
     }
 
-    var constructor = txHelper.getConstructorInterface(selectedContract.contract.object.abi)
-    self._deps.filePanel.compilerMetadata().metadataOf(selectedContract.name, (error, contractMetadata) => {
-      if (error) return self._deps.logCallback(`creation of ${selectedContract.name} errored: ` + error)
-      if (contractMetadata.autoDeployLib) {
-        txFormat.buildData(selectedContract.name, selectedContract.contract.object, self._deps.compiler.getContracts(), true, constructor, args, (error, data) => {
-          createInstanceCallback(error, selectedContract, data)
-        }, (msg) => {
-          self._deps.logCallback(msg)
-        }, (data, runTxCallback) => {
-          // called for libraries deployment
-          self._deps.udapp.runTx(data, runTxCallback)
-        })
-      } else {
-        txFormat.encodeConstructorCallAndLinkLibraries(selectedContract.contract.object, args, constructor, contractMetadata.linkReferences, selectedContract.contract.object.evm.bytecode.linkReferences, (error, data) => {
-          createInstanceCallback(error, selectedContract, data)
-        })
-      }
-    })
+    var forceSend = () => {
+      var constructor = txHelper.getConstructorInterface(selectedContract.contract.object.abi)
+      self._deps.filePanel.compilerMetadata().metadataOf(selectedContract.name, (error, contractMetadata) => {
+        if (error) return self._deps.logCallback(`creation of ${selectedContract.name} errored: ` + error)
+        if (contractMetadata.autoDeployLib) {
+          txFormat.buildData(selectedContract.name, selectedContract.contract.object, self._deps.compiler.getContracts(), true, constructor, args, (error, data) => {
+            createInstanceCallback(error, selectedContract, data)
+          }, (msg) => {
+            self._deps.logCallback(msg)
+          }, (data, runTxCallback) => {
+            // called for libraries deployment
+            self._deps.udapp.runTx(data, runTxCallback)
+          })
+        } else {
+          txFormat.encodeConstructorCallAndLinkLibraries(selectedContract.contract.object, args, constructor, contractMetadata.linkReferences, selectedContract.contract.object.evm.bytecode.linkReferences, (error, data) => {
+            createInstanceCallback(error, selectedContract, data)
+          })
+        }
+      })
+    }
+
+    if (selectedContract.contract.object.evm.deployedBytecode.object.length > 24576) {
+      modalDialog('Contract code size over limit', yo`<div>Contract creation initialization returns data with length of more than 24576 bytes. The deployment will likely fails.</div>`,
```

Comment:
could we add a link to the iep too?

Reply:
@yann300 -- i've added it ^^

Reply:
thanks but i meant as a link in the content of the modal too